src/Cell.tsx

```diff
@@ -1,38 +1,50 @@
-import React, { forwardRef, memo } from 'react';
+import React, { forwardRef, memo, useEffect, useRef } from 'react';
 import clsx from 'clsx';
 
 import { CellRendererProps } from './common/types';
-import { preventDefault, wrapEvent } from './utils';
+import { wrapEvent } from './utils';
+import { useCombinedRefs } from './hooks';
 
 function Cell<R, SR>({
   className,
   column,
+  isSelected,
+  isCopied,
+  isDraggedOver,
   isRowSelected,
   lastFrozenColumnIndex,
   row,
   rowIdx,
   eventBus,
+  dragHandle,
   onRowClick,
   onClick,
   onDoubleClick,
   onContextMenu,
-  onDragOver,
   ...props
 }: CellRendererProps<R, SR>, ref: React.Ref<HTMLDivElement>) {
+  const cellRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (isSelected) {
+      cellRef.current?.focus();
```

Comment:
This causes issues when scrolling, as we re-focus the cell when it reappears in the viewport, and the browser will try scrolling to it.

Reply:
Running the effect only on update now. This fixes the issue

Reply:
Fixed by rendering an row that contains the selected cell